src/odb/src/lefin/lefin.cpp

```diff
@@ -2221,17 +2230,40 @@ bool lefin::readLef(const char* lef_file)
     }
   }
   _incomplete_props.clear();
-  if (_layer_cnt)
-    _logger->info(
-        utl::ODB, 223, "    Created {} technology layers", _layer_cnt);
 
-  if (_via_cnt)
-    _logger->info(utl::ODB, 224, "    Created {} technology vias", _via_cnt);
-  if (_master_cnt)
-    _logger->info(utl::ODB, 225, "    Created {} library cells", _master_cnt);
+  std::string p = lef_file;
+  std::vector<std::string> parts;
+
+  if (_layer_cnt > 0) {
+    std::ostringstream ss;
+    ss << _layer_cnt << " layers";
+    parts.push_back(ss.str());
+  }
+
+  if (_via_cnt > 0) {
+    std::ostringstream ss;
+    ss << _via_cnt << " vias";
```

Comment:
Just FYI fmt::format is a nice replacement for this pattern.

Reply:
Is CentOS 7 phased out?

Reply:
Yes but how does that relate?

Reply:
I dumbed down this PR so it would work on centos

Reply:
We have been using it on centos7, eg

```
dbname = fmt::format("{}.main.{}.db", filename, getpid());
````